includes/utils.php

```diff
@@ -295,17 +310,30 @@ function distributable_post_types() {
 		unset( $post_types['attachment'] );
 	}
 
+	/**
+	 * Filter the post types that should be available for push.
+	 *
+	 * Helpful for sites that want to push custom post type content to another site.
+	 *
+	 * @hook dt_available_push_post_types
+	 *
+	 * @param {array} Post types that are distributable. Default all post types except `dt_ext_connection` and `dt_subscription`.
+	 *
+	 * @return {array} Post types available for push.
+	 */
+	$push_post_types = apply_filters( 'dt_available_push_post_types', array_diff( $post_types, [ 'dt_ext_connection', 'dt_subscription' ] ) );
```

Comment:
I think this will need to go in the `syndicate()` function used for the push-ui.

https://github.com/10up/distributor/blob/061383441a8b0dbc21a3b5fc4d2bf0063a14349a/includes/push-ui.php#L38

The final check within `syndicate()` checks the post types, so the new filter can be set up there.

Reply:
@peterwilsoncc `distributable_post_types` function is used in multiple places, so if we are adding this in `syndicatable()` function then other places won't get the right data.

Reply:
@peterwilsoncc that makes sense. I have moved that filter now.